src/explore-education-statistics-admin/src/pages/release/content/components/__tests__/ReleaseContentRedesign.test.tsx

```diff
@@ -63,4 +64,36 @@ describe('ReleaseContentRedesign', () => {
     tabPanels = screen.getAllByTestId('release-page-tab-panel');
     expect(tabPanels).toHaveLength(4);
   });
+
+  test('renders methodology tab content', async () => {
```

Comment:
Up to you, but this file could end up a bit large and unwieldy if the content for all the tabs is tested in it. It could be that it just tests the shell and each tab has its own test file for the content.

Also, to be a bit more thorough, it should probably test that it renders external and internal methodologies.